File: app/src/main/java/com/adaldosso/spendykt/api/BaseExpense.java
package com.adaldosso.spendykt.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Expense pojo
 *
 * Created by Alberto Dal Dosso on 02/10/2017.
 */

public class BaseExpense {

    @SerializedName("year")
    @Expose
    public String year;

    @SerializedName("month")
    @Expose
    public String month;

    @SerializedName("amount")
    @Expose
    public Float amount;

}

File: app/src/main/java/com/adaldosso/spendykt/api/BaseExpense.kt
package com.adaldosso.spendykt.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Expense pojo
 *
 * Created by Alberto Dal Dosso on 02/10/2017.
 */

open class BaseExpense {

    @SerializedName("month")
    @Expose
    open var month: String? = null

    @SerializedName("amount")
    @Expose
    var amount: Float? = null

    open var y: String? = null

    constructor(year: String) {
        this.y = year
    }

    open fun getYear(): String? {
        return y
    }
}